db_management.py

```diff
@@ -17,36 +17,45 @@ class User(Base):
     Password = Column(String, nullable=False, unique=True)
 
 
-def create_connection():
+def create_tables():
     Base.metadata.create_all(engine)
 
 
-def check_if_user_exists(str_username, str_pass):
+def create_connection():
+    engine = create_engine('sqlite:///data_base.db')
```

Comment:
Why don't you use the value in the configuration file?

Reply:
changed accordingly